spec/models/inventory_spec.rb

```diff
@@ -23,4 +26,43 @@
 
```

Comment:
We missed the tests where we expect to have many facilitators and many participants.